scico/test/test_numpy.py

```diff
@@ -48,6 +48,7 @@ def test_reshape_array():
     np.testing.assert_allclose(b.ravel(), c.ravel())
 
 
+@pytest.mark.skip  # do we care to support svd of matrix operator?
```

Comment:
I assume the intent is to discuss these questions and permanently remove the tests in case of affirmative decision?